src/site/_includes/content/lighthouse-best-practices/scoring.njk

```diff
@@ -1,8 +1,5 @@
-## How this audit impacts overall Lighthouse score
-
-Lighthouse returns a Best Practices score between 0 and 100.
-0 is the worst possible score, and 100 is the best.
-The Best Practices audits are equally weighted.
-To calculate how much each audit contributes
-to your overall Best Practices score,
-count the number of Best Practices audits, then divide 100 by that number.
\ No newline at end of file
+{% Aside 'note' %}
```

Comment:
Linking back to /web for these asides doesn't quite feel right to me either.

Would it make sense to add a scoring doc to the end of each learning path and link to that instead?

Reply:
Totally do-able to create short docs for SEO, Best Practices, and PWA. I'm presuming that Fast and Accessibility are their own universes given the fact that the scoring differs per audit. We could though create an issue to revisit these areas too post-IO, as a more thoughtful scoring guide for each of these could be good. It's just a time factor at this stage.

Reply:
I haven't put a ton of thought into it buuuuuuut... it does seem like it might be ok to have a general scoring doc for seo, best practices, and pwa. And then have inline scoring explanations for accessibility and performance. If we did want to be really consistent we could probably draft general scoring docs for accessibility and performance which explain, at a high level, how their scoring works, and then informs the reader that more specific guidance will be offered in the audit docs themselves?

Reply:
For the sake of speed, I'm gonna add docs for PWA, SEO, and best practices, but not perf and accessibility at this time.

But I will create separate issues to do this task post-IO-- one for accessibility scoring and one for perf scoring.

Reply:
As mentioned previously, my approach for now is to add docs to best practices, seo, and pwa.

And then I'm creating issues for accessibility and perf scoring to work on post-IO.